core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -105,59 +102,41 @@ public class LocalCacheManager implements CacheManager {
   /**
    * @param conf the Alluxio configuration
    * @param metaStore the metadata store for local cache
+   * @param pageStoreDirs the list of the directory for local cache
    * @return an instance of {@link LocalCacheManager}
    */
-  public static LocalCacheManager create(AlluxioConfiguration conf, MetaStore metaStore)
-      throws IOException {
-    PageStoreOptions options = PageStoreOptions.create(conf);
-    PageStore pageStore;
-    try {
-      pageStore = PageStore.open(options);
-    } catch (IOException e) {
-      pageStore = PageStore.create(options);
-    }
-    return create(conf, metaStore, pageStore);
-  }
-
-  /**
-   * @param conf the Alluxio configuration
-   * @param metaStore the meta store manages the metadata
-   * @param pageStore the page store manages the cache data
-   * @return an instance of {@link LocalCacheManager}
-   */
-  @VisibleForTesting
-  static LocalCacheManager create(AlluxioConfiguration conf, MetaStore metaStore,
-      PageStore pageStore) throws IOException {
-    LocalCacheManager manager = new LocalCacheManager(conf, metaStore, pageStore);
-    PageStoreOptions options = PageStoreOptions.create(conf);
+  public static LocalCacheManager create(AlluxioConfiguration conf, MetaStore metaStore,
+                                         List<PageStoreDir> pageStoreDirs) throws IOException {
```

Comment:
nits: i don't think the format is correct here.

Reply:
fixed, I guess I might check the "align when multi-line" by mistake. thanks!